frontend/src/components/Navigation.tsx

```diff
@@ -63,7 +64,7 @@ export const Navigation = injectIntl(({ intl }): JSX.Element => {
               { intl.formatMessage({ id: 'teams' }) }
             </Link>
           </Menu.Item>
-          { userRoles.includes('manager')
+          { rolesCheck(userRoles, ['manager', 'supervisor_editor'])

```

Comment:
Nie sprawdzaj obecności roli `manager`. To tylko rola techniczna, która agreguje inne role, po to aby można było przekazać użytkownikowi wiele uprawnień jednocześnie. Ta agregacja jest już w keycloak i nie powinno jej być ani w backendzie ani we frontendzie.